schema.graphql

```diff
@@ -92,12 +93,37 @@ type AssessmentDetail {
   id: ID!
   assessment: Assessment!
   assessmentMetrics: [AssessmentMetric!] @hasInverse(field: assessmentDetail)
-  verseTexts: [VerseText]
+  verseTexts: [VerseText!]
+  startPosition: Int
+  endPosition: Int
+  refVerseTexts: [VerseText]
+  refStartPosition: Int
+  refEndPosition: Int
+  annotations: [Annotation] @hasInverse(field: assessmentDetail)
 }
 
 type AssessmentMetric {
   id: ID!
   assessmentDetail: AssessmentDetail!
   flag: Boolean @search
   number: Float @search
+  tag: String @search
 }
+
+enum AnnotationType {
+  SubAssessment
+  VersePart
+  PartOfSpeech
+}
+
+type Annotation {
+  id: ID!
+  verseTexts: [VerseText!]
+  startPosition: Int
```

Comment:
Is there duplication between this span in the annotation and the span represented in the assessment detail?